socorrolib/lib/migrations.py

```diff
@@ -12,6 +12,8 @@
 import os
 import warnings
 
+import socorro
```

Comment:
This creates a circular dependency problem where socorro relies on socorrolib and socorrolib relies on socorro. At that point, why even have a socorrolib?

I think if we're going to do this, then we should switch gears and instead pull all the socorrolib stuff back into socorro.